jwst/jump/twopoint_difference.py

```diff
@@ -53,18 +52,22 @@ def find_CRs(data, gdq, read_noise, rej_threshold, nframes):
 
         # Compute first differences of adjacent groups up the ramp
         first_diffs = np.diff(rdata, axis=2)
-
-        # Compute median of the first differences for all pixels
-        med_diffs = np.nanmedian(first_diffs, axis=2)
-
-        # Zero-out results for pixels that have NaN's in all groups so they
-        # don't cause trouble in later calculations
-        nans = np.where(np.isnan(med_diffs))
-        med_diffs[nans] = 0.
-        first_diffs[nans] = 0.
-
         nans = np.where(np.isnan(first_diffs))
-        first_diffs[nans] = 0.
+        first_diffs[nans] = 100000.
+
+        print(repr(first_diffs[100,100,:]))
```

Comment:
All print statements will need to either be removed for use within the pipeline environment or converted to logging calls. If it's useful to keep the statements for debugging work, then they can be converted to calls to "log.debug" so that they appear only when run with the logging level set to debug by the user.